coursebook/index.md

```diff
@@ -1,7 +1,6 @@
 # Welcome!
 
-Welcome to an **Introduction to Research Data Science**, developed by
-developed by [The Alan Turing Institute's](https://www.turing.ac.uk/) 
+Welcome to an **Introduction to Research Data Science**, developed by [The Alan Turing Institute's](https://www.turing.ac.uk/) 
```

Comment:
But what is Research Data Science?

Reply:
Commits https://github.com/alan-turing-institute/rds-course/pull/138/commits/617001ddeac94dff7f222548cc6db14ab9cf6596 and https://github.com/alan-turing-institute/rds-course/pull/138/commits/575e4c606182e5b146d21fe574ec569cee26324b tweak this slightly - basically shifted a disclaimer too. Thought this was better to have on index to make it very clear that we are biased in some definitions and do not claim to be the definitive authority